src/applications/static-pages/renderHomepageBanner.js

```diff
@@ -0,0 +1,51 @@
+import yaml from 'js-yaml';
```

Comment:
it seems like this is deliberately a short-term hack to get something out asap for this weekend. can you please document this file very clearly and loudly as such? don't want this code to be seen as an acceptable pattern until we've gotten a chance to talk through it more + evaluate other ideas

Reply:
There is a CMS requirement to move this banner's configuration into there, so this approach will be retired as part of that implementation.

Reply:
I agree it'll have an end-date, I'm not worried about forgetting and leaving this here --- I'm more concerned about someone who's out of this loop poking around the codebase, seeing this code checked in, and thinking that this is an allowed approach that we've validated.

so I think we should still put a warning comment in here to basically indicate, "hey we know this isn't OK, don't copy this"

Reply:
Added a comment here telling folks not to repeat this - https://github.com/department-of-veterans-affairs/vets-website/pull/10665/files#diff-9e32aae52d3cace003ef8ce877cd08d6R28